spec/models/book_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe Book do
+  subject(:book) { create(:book) }
+
+  it '#validate' do
+    is_expected.to validate_presence_of :genre
+    is_expected.to validate_presence_of :author
+    is_expected.to validate_presence_of :image
+    is_expected.to validate_presence_of :title
+    is_expected.to validate_presence_of :publisher
+    is_expected.to validate_presence_of :year
+  end
+
+  it 'valid book' do
+    is_expected.to be_valid
```

Comment:
for this basic tests it is not necessary to add a description, you can just put
`it { is_expected.to be_valid }` and that's enough :)

Reply:
ohh Good.. I didn't know